CHANGELOG.rst

```diff
@@ -134,6 +134,7 @@ Removed
   ``.from_dict()`` directly (#1360).
 - The ``qiskit.Result`` class method for ``len()`` and indexing have been
   removed, along with the functions that perform post-processing (#1351).
+- The ``get_snapshots`` from the ``result object.
```

Comment:
I would add to the end here something like: "Instead you can access the snapshots in the Result from `Result.data()['snapshots']"

Reply:
I pushed https://github.com/Qiskit/qiskit-terra/pull/1380 to update this